src/ekat/io/ekat_yaml.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef EKAT_YAML_HPP
+#define EKAT_YAML_HPP
+
+#include "ekat/ekat_parameter_list.hpp"
+#include <string>
+
+namespace ekat {
```

Comment:
I think we should mention that vectors of booleans are stored as `vector<char>` here, for lack of a better place to warn folks against `vector<bool>`.